building_with_meson.md

```diff
@@ -1,7 +1,7 @@
 # Building with Meson
 
 _Note: this is for early adopters. It has been tested on Linux and macOS, and
-with Python 3.9-3.12. Windows will be tested soon. There is one CI job to keep
+with Python 3.10-3.12. Windows will be tested soon. There is one CI job to keep
```

Comment:
We definitely support windows now. Is there really only one CI job? I think the whole CI now uses meson. Maybe this whole paragraph should be deleted.
```suggestion
with Python 3.10-3.12. There is one CI job to keep
```